web/app/themes/clarity/inc/admin/acf-field-group.php

```diff
@@ -5951,26 +5899,6 @@
 			'append' => '',
 			'maxlength' => '',
 		),
-		array(
```

Comment:
why is agency_wide_banner_message being removed

Reply:
👍